src/odemis/acq/feature.py

```diff
@@ -142,7 +142,11 @@ def load_project_data(path: str) -> dict:
     # load feature streams
     for f in features:
         # search dir for images matching f.name.value
-        stream_filenames = glob.glob(os.path.join(path, f"*{f.name.value}*.ome.tiff"))
+        stream_filenames = []
+        glob_path = os.path.join(path, f"*-{glob.escape(f.name.value)}-{{ext}}")
+        for ext in ["*.ome.tif*", "*.h5"]:
```

Comment:
"ome.tif*" can match some odd texts too. I'd write (to also be compatible with "ij.tiff"):
```suggestion
        for ext in ["*.tif", "*.tiff", "*.h5"]:
```